arches/app/views/tile.py

```diff
@@ -88,6 +88,9 @@ def handle_save_error(self, e, tile_id="", title=_("Saving tile failed"), messag
         return JSONErrorResponse(_(title), _(str(message)), {"message": message, "title": title})
 
     def post(self, request):
+        # TODO: accept UUID, don't generate here.
+        transaction_id = uuid.uuid1()
```

Comment:
I think that here we can simply do something like this:
```transaction_id = request.POST.get("transaction_id", uuid.uuid1())```
in order to retrieve the transaction id from the request where needed.